### PR TITLE
v0.19.0 release notes

### LINKED ISSUE #5588: Release v0.19?
Yesterday in the dev call we discussed the need for another release. Not sure if this should be a bugfix release (i.e. v0.18.3) or a full release (i.e. v0.19). Last release (v0.18.2) was 19th May, with v0.18.0 on 6th May.

@pydata/xarray

Bug fixes:

- #5581 and the fix #5359 (this one needs to be released soon really)
- #5528
- Probably various smaller ones

New features:

- #4696
- #5514
- #5476
- #5464
- #5445

Internal:
- `master` -> `main` #5520
- #5506

Nice to merge first?:

- [x] #5568 and #5561
- [ ] #5571
- [x] #5586
- [ ] #5493
- [x] #4909
- [ ] #5580
- [ ] #4863
- [ ] #5501

### PR DESCRIPTION
**Pull request by TomNicholas**
Release notes: 

```rst
This release brings improvements to plotting of categorical data, the ability to specify how attributes
are combined in xarray operations, a new high-level :py:func:`unify_chunks` function, as well as various
deprecations, bug fixes, and minor improvements.
```

- [x] Closes #5588

### CODE REVIEW

**Comment by TomNicholas** [comment]
Confused as to how this has extra commits in it, but the only file changed is the what's new, so I think it's fine.

**Comment by keewis** [comment]
I cherry-picked the "fix RTD" commit to `main` so that shouldn't introduce any new changes (apparently, git doesn't detect that). The revert commit and the release notes for 0.18.2 are important, though, we don't want to lose those.

File: doc/whats-new.rst [hunk 1]
```diff
@@ -14,11 +14,24 @@ What's New
 
     np.random.seed(123456)
 
-.. _whats-new.0.18.3:
 
-v0.18.3 (unreleased)
+.. _whats-new.0.19.0:
+
+v0.19.0 (23 July 2021)
 ---------------------
```
Comment (keewis) [inline doc/whats-new.rst:21]:
```suggestion
----------------------
```

**Comment by TomNicholas** [comment]
Is that an approval then @keewis ?

**Comment by keewis** [comment]
yes, it is. I would do something about the empty "Documentation" section (and the empty sections in 0.18.1), though. Remove it, maybe?

**Pull request merged**